common/plugins/eu.esdihumboldt.hale.common.instance/src/eu/esdihumboldt/hale/common/instance/io/util/EnumWindingOrderTypes.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016 Data Harmonisation Panel
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     Data Harmonisation Panel <http://www.dhpanel.eu>
+ */
+
+package eu.esdihumboldt.hale.common.instance.io.util;
+
+/**
+ * TODO Type description
+ * 
+ * @author Arun
+ */
+public enum EnumWindingOrderTypes {
+	CLOCKWISE, COUNTERCLOCKWISE, DONTCHANGE;
```

Comment:
As the enum names are also the parameter values to be provided to a writer I would prefer names that are not all uppercase. The parameter values for similar settings usually use camel case.
So the settings could be `clockwise`, `counterClockwise` and `noChanges`.

When you create a pull request, please try to eliminate all compiler warnings in Eclipse that are introduced by your changes. I'm quite sure that there are warnings here about missing documentation.
This would be the perfect spot to document the different modes and state that the winding order is applied in reverse to polygon holes.